### PR TITLE
DX-2141

### PR DESCRIPTION
**Pull request by ajrice6713**
Add Object property descriptions

## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

File: site/specs-source/dash.json [hunk 1]
```diff
@@ -457,259 +457,329 @@
...
           }
         }
       },
       "Location": {
         "type": "object",
         "properties": {
           "activatedtime": {
-            "type": "string"
+            "type": "string",
+            "description": "Time  that the most recent address was activated (provisioned) for the endpoint (in UTC)"
```
Comment (hamermike) [inline site/specs-source/dash.json:494]:
```suggestion
            "description": "Time that the most recent address was activated (provisioned) for the endpoint (in UTC)"
```

File: site/specs-source/dash.json [hunk 2]
```diff
@@ -734,10 +804,12 @@
...
                 "properties": {
                   "callername": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Name of subscriber for the endpoint"
                   },
                   "uri": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Effectively the phone number of the subscriber.  The URI can include the \"tel:\" prefix, but it's not required."
```
Comment (hamermike) [inline site/specs-source/dash.json:812]:
```suggestion
                    "description": "Effectively the phone number of the subscriber. The URI can include the \"tel:\" prefix, but it's not required."
```

File: site/specs-source/dash.json [hunk 3]
```diff
@@ -797,10 +869,12 @@
             "type": "object",
             "properties": {
               "uri": {
-                "type": "string"
+                "type": "string",
+                "description": "Effectively the phone number of the subscriber.  The URI can include the \"tel:\" prefix, but it's not required."
```
Comment (hamermike) [inline site/specs-source/dash.json:873]:
```suggestion
                "description": "Effectively the phone number of the subscriber. The URI can include the \"tel:\" prefix, but it's not required."
```

File: site/specs-source/dash.json [hunk 4]
```diff
@@ -457,259 +457,329 @@
...
           }
         }
       },
       "ProvisionedLocations": {
         "type": "object",
         "properties": {
           "activatedtime": {
-            "type": "string"
+            "type": "string",
+            "description": "Time  that the most recent address was activated (provisioned) for the endpoint (in UTC)"
```
Comment (hamermike) [inline site/specs-source/dash.json:692]:
```suggestion
            "description": "Time that the most recent address was activated (provisioned) for the endpoint (in UTC)"
```

**Review by hamermike — CHANGES_REQUESTED** [review]

**Review by hamermike — APPROVED** [review]

**Review by jmulford-bw — APPROVED** [review]
Approving assuming there will be future changes to fix the examples

**Pull request merged**